static/swaggerui/swagger.json

```diff
@@ -0,0 +1,456 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "This is the first version of Kube-green API.",
+    "version": "1.0.0",
+    "title": "Kube-green API",
+    "termsOfService": "http://swagger.io/terms/",
+    "contact": {
+      "email": "apiteam@swagger.io"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "host": "",
+  "basePath": "/v1",
```

Comment:
If basePath indicates the main path the backend is hosted from, this is just `/` not `/v1` or something